docs/content/usages/assertions.md

```diff
@@ -0,0 +1,250 @@
+---
+sort: 6
+---
+
+# Assertions
+
+Assertions are a set of action properties available to every action to assert the result after its execution.
+
+Assertions are executed after a handler and an error is thrown if the assertion doesn't validate. The name of the properties is prefixed with `assert` or `unassert` for their negation.
```

Comment:
They validate the value returned by the handler function. On failure, an error is thrown with the `NIKITA_INVALID_ASSERTION` code. The names (typo) of the properties are prefixed with `assert` or `unassert` for their negation.